Detectors/gconfig/g3libs.C

```diff
@@ -20,11 +20,11 @@ void g3libs()
 {
   cout << "Loading Geant3 libraries ..." << endl;
 
-  if (isLibrary("libdummies.so"))
+  if (isLibrary("libdummies"))
      gSystem->Load("libdummies.so");
```

Comment:
These are dummy implementations of some Fortran functions, so the name corresponds to the content :)